configs/stac/demo.yaml

```
@@ -1,10 +1,13 @@
-fit_path: "demo_fit.p"
+fit_offsets_path: "demo_fit.p"
 ik_only_path: "demo_ik_only.p"
 data_path: "tests/data/test_rodent_mocap_1000_frames.mat"
+fit_offsets_path: "synth_fit.p"
```

yamllint

[error] duplication of key "fit_offsets_path" in mapping

(key-duplicates)